docs/conf.py

```diff
@@ -0,0 +1,166 @@
+# -*- coding: utf-8 -*-
+import os
+import sys
+import pathlib
+import datetime
+
+from pkg_resources import get_distribution
+
+try:
+    from sphinx_astropy.conf.v1 import *  # noqa
+except ImportError:
+    print('ERROR: the documentation requires the sphinx-astropy package to be installed')
+    sys.exit(1)
+
+# Get configuration information from setup.cfg
+from configparser import ConfigParser
+conf = ConfigParser()
+
+conf.read([os.path.join(os.path.dirname(__file__), '..', 'setup.cfg')])
+setup_cfg = dict(conf.items('metadata'))
+
+# -- General configuration ----------------------------------------------------
+
+# By default, highlight as Python 3.
+highlight_language = 'python3'
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+exclude_patterns.append('_templates')
+
+# This is added to the end of RST files - a good place to put substitutions to
+# be used globally.
+rst_epilog += """
+"""
+
+# -- Project information ------------------------------------------------------
+
+# This does not *have* to match the package name, but typically does
+project = setup_cfg['name']
+author = setup_cfg['author']
+copyright = '{0}, {1}'.format(
+    datetime.datetime.now().year, setup_cfg['author'])
+copyright = ''
+
+release = get_distribution(setup_cfg['name']).version
+version = '.'.join(release.split('.')[:3])
+
+
+# -- Options for HTML output --------------------------------------------------
+
+# A NOTE ON HTML THEMES
+# The global astropy configuration uses a custom theme, 'bootstrap-astropy',
+# which is installed along with astropy. A different theme can be used or
+# the options for this theme can be modified by overriding some of the
+# variables set in the global configuration. The variables set in the
+# global configuration are listed below, commented out.
+
+
+# Add any paths that contain custom themes here, relative to this directory.
+# To use a different custom theme, add the directory containing the theme.
+#html_theme_path = []
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes. To override the custom theme, set this to the
+# name of a builtin theme or the name of a custom theme in html_theme_path.
+#html_theme = None
+
+
+html_theme_options = {
+    'logotext1': 'roentgen',  # white,  semi-bold
+    'logotext2': '',  # orange, light
+    'logotext3': ':docs',   # white,  light
+    'astropy_project_menubar': False
+    }
+
+
+# Custom sidebar templates, maps document names to template names.
+#html_sidebars = {}
+
+# The name of an image file (relative to this directory) to place at the top
+# of the sidebar.
+html_logo = 'logo/roentgen_logo.jpg'
+
+# The name of an image file (within the static path) to use as favicon of the
+# docs.  This file should be a Windows icon file (.ico) being 16x16 or 32x32
+# pixels large.
+#html_favicon = ''
```

Comment:
DId you mean to use the default astropy theme for the docs with their favicon?

Reply:
moving to sunpy theme